### ISSUE TITLE
Container vs. No Container

### ISSUE THREAD

**Issue by jordanwillis** [paularrj/teacozy#1]
First off, you did a fantastic job on this project!  My review comments typically include things like how to correctly use wrapping with flexbox, proper HTML structuring, missing `alt` attributes, etc., but in this case the project is essentially perfect.  Therefore, I will provide some more general feedback and recommendations of things for you to think about in the future.

One thing that I noticed about your HTML style is the tendency to always create containers even when not completely necessary.  For example...

https://github.com/paularrj/teacozy/blob/0167cc47421cc0c6b695def685b01bcca6bb3922/index.html#L11-L13

https://github.com/paularrj/teacozy/blob/0167cc47421cc0c6b695def685b01bcca6bb3922/index.html#L31-L34

There is absolutely nothing wrong with this approach and many times adding the extra structure further enhances the semantics of your HTML, but I wanted to highlight that it is not always necessary.  The rule of thumb that I tend to follow is to use a container when the group of elements act together as a single component or when I need to apply styling to the elements within a certain context.  Otherwise, I do not add a container to ensure that my code is as simplified as possible.

So for example, your logo section could have done without the container allowing the `<img>` element to stand alone.

``` html
<header class="flex-container">
  <img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-4/img-tea-cozy-logo.png" />
  <nav>
    <span><a href="#mission">Mission</a></span>
    <span><a href="#store">Featured Tea</a></span>
    <span><a href="#locations">Locations</a></span>
  </nav>
</header>
```

In my opinion it just makes things a bit cleaner and prevents having to create CSS selectors like this (which typically indicate that the container isn't necessary....because the `logo` class is really never directly used).

https://github.com/paularrj/teacozy/blob/0167cc47421cc0c6b695def685b01bcca6bb3922/css/style.css#L23-L26

Whereas in my example above, the image styling is done from the context of the `<header>` (instead of an additional container).

``` css
header img {
  height: 50px;
  padding-left: 10px;
}
```

Long story short, I guess what I'm recommending is to make a solid judgment call for when a container is actually necessary vs. when you are creating them just by default.
